frontend/src/components/Collections/components/Grid/components/Row/DatasetRow/utils.ts

```diff
@@ -55,3 +89,218 @@ export function checkIfCancelled(datasetStatus: DatasetUploadStatus): boolean {
     datasetStatus.upload_status === UPLOAD_STATUS.CANCELED
   );
 }
+
+export function useCheckCollectionPopulated({
```

Comment:
both `useCheckCollectionPopulated` and `useCheckCollectionFormatsPopulated ` use the same helper hook `useCheckCollection` under the hood